include/IGameLogic.hpp

```diff
@@ -0,0 +1,26 @@
+/*
+** EPITECH PROJECT, 2023
+** Visual Studio Live Share (Workspace)
+** File description:
+** ISystem
+*/
+
+#pragma once
+
+#include <cstddef>
+#include <vector>
+#include "IEntity.hpp"
+#include "IEventManager.hpp"
+#include "ISystem.hpp"
+
+namespace Arcade {
+    namespace Game {
+        class IGameLogic : Arcade::ECS::ISystem {
```

Comment:
No doc for this class, i don t know if it's necessary but it could be nice

Reply:
I don't know what to say in the doc for this